params/licenses.go

```diff
@@ -0,0 +1,37 @@
+package params
+
+import (
+	_ "embed"
+
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+type DepLicense struct {
+	Name    string  `json:"name"`
+	Version *string `json:"version"`
+	Type    string  `json:"type"`
+	Path    string  `json:"path"`
+}
+
+var (
+	// Polygon Edge License
+	License string
+
+	// Dependency Licenses
+	//go:embed bsd_licenses.json
+	bsdLicensesJSON string
+	BsdLicenses     []DepLicense
```

Comment:
These acronyms should always be upper-cased, like `BSDLicenses` and so on

Reply:
Renamed: 236f035318fbc226dd45dc6962a7bbc88e7b642f